src/core/LanguageMenu/LanguageMenuItem/LanguageMenuItem.tsx

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import classnames from 'classnames';
-import {
-  MenuLink as CompMenuLink,
-  LanguageMenuLinkProps,
-} from '../../components/LanguageMenu/LanguageMenu';
+import { MenuItem } from '@reach/menu-button';
+import { LanguageMenuItemBaseProps } from '../LanguageMenu';
 
-export interface LanguageMenuLinkLanguageProps extends LanguageMenuLinkProps {
+export interface LanguageMenuItemProps extends LanguageMenuItemBaseProps {
```

Comment:
It seems that GitHub shows wrong files in the compare, because of the folder structure change and rename. Also same case with the `LanguageMenuLink` file below.